lib/ansible/modules/database/mysql/mysql_db.py

```diff
@@ -110,6 +110,12 @@
     required: no
     type: bool
     default: no
+  dump_extra_args:
+    description:
+      - Provide additional arguments for mysqldump.
+        Used when I(state=dump) only, ignored otherwise.
+    required: false
+    type: str
     version_added: '2.10'
```

Comment:
That's now missing above (for `skip_lock_tables`).

Also, please don't add merge commits but rebase the PR instead (see link in the ansibot message if you don't know how to do it).